.betterer.results

```diff
@@ -3805,11 +3805,12 @@ exports[`better eslint`] = {
     ],
     "public/app/features/explore/state/utils.ts:5381": [
       [0, 0, 0, "Do not use any type assertions.", "0"],
-      [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
-      [0, 0, 0, "Do not use any type assertions.", "2"],
-      [0, 0, 0, "Unexpected any. Specify a different type.", "3"],
-      [0, 0, 0, "Do not use any type assertions.", "4"],
-      [0, 0, 0, "Do not use any type assertions.", "5"]
+      [0, 0, 0, "Do not use any type assertions.", "1"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "2"],
+      [0, 0, 0, "Do not use any type assertions.", "3"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "4"],
+      [0, 0, 0, "Do not use any type assertions.", "5"],
+      [0, 0, 0, "Do not use any type assertions.", "6"]
```

Comment:
This was added because we need a utility function that checks if string is enum in `for (const string of Object.keys(enum))` to satisfy typescript.

Reply:
This was removed in https://github.com/grafana/grafana/pull/60998/commits/2c7b92ea0ec7a9f14049161befe6ad5ca8824f53 by using `const type of SUPPORTING_QUERY_TYPES`